### DISCUSSION TITLE
sealos部署k8s必须使用labring仓库中的镜像么，labring中的镜像修改了什么？

### DISCUSSION THREAD

**Discussion by ttsdmmn** [labring/sealos#4957] [Q&A]
sealos部署k8s必须使用labring仓库中的镜像么，labring中的镜像修改了什么？

**Comment by bxy4543** [comment]
sealos使用的是集群镜像，可以参考文档：https://sealos.io/zh-Hans/docs/self-hosting/lifecycle-management/reference/sealos/
